man/scrot.txt

```diff
@@ -4,7 +4,7 @@ NAME
 SYNOPSIS
   scrot [-bcfhimopuvz] [-a X,Y,W,H] [-C NAME] [-D DISPLAY] [-d SEC] [-e CMD]
         [-F FILE] [-k OPT] [-l STYLE] [-M NUM] [-n OPTS] [-q NUM] [-S CMD] [-s OPTS]
-        [-t NUM | GEOM] [FILE]
+        [-t PCT | WxH] [FILE]
```

Comment:
`PCT` is very awkward. I'd either spell it out fully or use `%`.

Reply:
% seems fine to me, I will change it.